docs/podtemplates.md

```diff
@@ -95,6 +95,40 @@ Pod templates support fields listed in the table below.
 	</tbody>
 </table>
 
+## Use `imagePullSecrets` to lookup entrypoint
+
+If no command is configured in `task` and `imagePullSecrets` is configured in `podTemplate`, the Tekton Controller will look up the entrypoint of image with `imagePullSecrets`. The Tekton controller's service account is given access to secrets by default. See [this](https://github.com/tektoncd/pipeline/blob/0040bcaa9ab911b8b141699e94fee303e2134f63/config/200-clusterrole.yaml#L58-L61) for reference. If the Tekton controller's service account is not granted the access to secrets in different namespace. You need to grant the access via RoleBinding:
```

Comment:
nit: Can we link to a recent released tag instead of a random commit? Or maybe just link to `main` so we don't have to keep updating it?

Reply:
sure! I will fix it.